recipes/emscripten-activation/activate.sh

```diff
@@ -0,0 +1,62 @@
+if [ -z ${CONDA_FORGE_EMSCRIPTEN_ACTIVATED+x} ]; then
+
+    export CONDA_FORGE_EMSCRIPTEN_ACTIVATED=1
+
+    export EMSDK_PYTHON=${CONDA_PREFIX}/bin/python3
+    export PYTHON=${CONDA_PREFIX}/bin/python3
+
+    CONDA_EMSDK_DIR=$CONDA_PREFIX/opt/emsdk
+ 
+    export EMSCRIPTEN_VERSION=$PKG_VERSION
+    export EMSCRIPTEN_FORGE_EMSDK_DIR=$CONDA_EMSDK_DIR
+
+    # clear all prexisting cmake args / CC / CXX / AR / RANLIB
+    export CC="emcc"
+    export CXX="em++"
+    export AR="emar"
+    export RANLIB="emranlib"
+    
+    export CMAKE_ARGS=""
+
+    # set the emscripten toolchain
+    export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_TOOLCHAIN_FILE=$CONDA_EMSDK_DIR/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake"
+
+    # conda prefix path
+    export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_PREFIX_PATH=$PREFIX"
+
+    # install prefix
+    export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=$PREFIX"
+
+    # find root path mode package
+    export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_FIND_ROOT_PATH_MODE_PACKAGE=ON"
+
+    # fpic
+    export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=true"
+
+    cmake () {
+        emcmake cmake "$@"
+    }
+
+    # useful variables
+    export PY_SIDE_LD_FLAGV
```

Comment:
What is the use of this variable?

Reply:
@DerThorsten, could you comment?

Reply:
this is very likely some leftover from some debugging in the past, should be removable